src/template-parameters.json

```diff
@@ -479,35 +469,15 @@
             "paramName": "method",
             "type": "EQUALS",
             "paramValue": "convertedObjectIDsAfterSearch"
-          },
-          {
-            "paramName": "method",
-            "type": "EQUALS",
-            "paramValue": "viewedObjectIDs"
           }
         ]
       },
       {
-        "type": "SIMPLE_TABLE",
+        "type": "TEXT",
```

Comment:
Forgive my ignorance here, but does a field of type "TEXT" allow the user to provide a variable to set a field within objectData?

For example if I store the `queryID` in a GTM variable, do I have access to that in this field?

Reply:
You can use literally _any_ variable that you define.

Reply:
The idea is that the customer would assign this object data info in a JS variable (or something similar) and read it using this field. This seems to be what other GTM tags do in order to read/write more complex data structures.